File: aspnet-core/src/Lesson.Application/Domain/ClassRoom/Dto/CreateClassRoomInput.cs
﻿using Lesson.Authorization.Users;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lesson.Domain.Dto
{
    public class CreateClassRoomInput
    {
        public string Name { get; set; }
        public string Branch { get; set; }
        public string Description { get; set; }
        public List<Lesson.Dto.LessonFullOutPut> Lessons { get; set; }
        public List<UserPartOutPut> Students { get; set; }

    }
}

File: angular/src/app/services/ClassRoom/dtos/CreateClassRoomInput.ts
import { UserPartOutPut } from './../../User/dtos/UserPartOutPut';
import { LessonFullOutPut } from './../../Lesson/dtos/LessonFullOutPut';
 export class CreateClassRoomInput implements ICreateClassRoomInput{
     name: string;
     branch: string;
     description: string;
     lessons: LessonFullOutPut[];
     students:UserPartOutPut[];
     constructor(data?: ICreateClassRoomInput) { }
 }

export interface ICreateClassRoomInput {
    name: string | undefined;
    branch: string | undefined;
    description: string | undefined;
    lessons:LessonFullOutPut[]|undefined;
    students:UserPartOutPut[]|undefined;
}
